src/control/fix.lean

```diff
@@ -0,0 +1,332 @@
+/-
+Copyright (c) 2020 Simon Hudon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Simon Hudon
+-/
+
+import data.nat.up
+import data.stream.basic
+import logic.basic
+import order.basic
+import order.bounded_lattice
+import order.omega_complete_partial_order
+import tactic.apply
```

Comment:
```suggestion
import data.nat.up
import data.stream.basic
import order.omega_complete_partial_order
```
none of these are needed for this file, and afaict not for transitive imports either